Source/WebCore/animation/AnimationTimelinesController.h

```diff
@@ -68,21 +68,21 @@ class AnimationTimelinesController final : public CanMakeCheckedPtr<AnimationTim
     bool animationsAreSuspended() const { return m_isSuspended; }
 
     void registerNamedScrollTimeline(const AtomString&, Element&, ScrollAxis);
-    void unregisterNamedScrollTimeline(const AtomString&);
-    ScrollTimeline* scrollTimelineForName(const AtomString&) const;
-
     void registerNamedViewTimeline(const AtomString&, Element&, ScrollAxis, ViewTimelineInsets&&);
-    void unregisterNamedViewTimelineForSubject(const AtomString&, const Element&);
-    ViewTimeline* viewTimelineForNameAndSubject(const AtomString&, const Element&) const;
+    void unregisterNamedTimeline(const AtomString&, Element&);
+    RefPtr<AnimationTimeline> timelineForName(const AtomString&, Element&) const;
 
 private:
     ReducedResolutionSeconds liveCurrentTime() const;
     void cacheCurrentTime(ReducedResolutionSeconds);
     void maybeClearCachedCurrentTime();
 
+    RefPtr<AnimationTimeline> timeline(Vector<Ref<AnimationTimeline>>, Element&) const;
```

Comment:
I think I'd just do away with that method and just have it inside `timelineForName` as it appears to have a single call site. If not, it should be `static` and ideally have a less generic name.

Reply:
No longer relevant for this patch.